scripts/globals/regimes.lua

```diff
@@ -1442,7 +1442,7 @@ xi.regime.checkRegime = function(player, mob, regimeId, index, regimeType)
     end
 
     -- award XP every page completion
-    player:addExp(reward)
+    player:addExp(reward * FOV_EXP_RATE)
```

Comment:
Did you test this? Settings live in the `xi.settings` namespace now, so this is `nil`

Reply:
oh sorry I typoed it in the PR, let me fix